.circleci/config.yml

```diff
@@ -5,38 +5,54 @@ references:
     GOMAXPROCS: 1
     NOMAD_SLOW_TEST: 1
     GOTESTSUM_JUNITFILE: /tmp/test-reports/results.xml
+  ignore_for_ui_branches: &IGNORE_FOR_UI_BRANCHES
+    filters:
+      branches:
+        ignore: /^.-ui\b.*/
+
 
 workflows:
   build-test:
     jobs:
-      - lint
+      - lint:
```

Comment:
we can rename this to `lint-go` to avoid confusion maybe?

Reply:
👍 will do, thanks!